src/Shared/Microsoft.DotNet.Services.Utility/Microsoft.DotNet.Services.Utility.csproj

```diff
@@ -7,5 +7,8 @@
   <ItemGroup>
     <PackageReference Include="WindowsAzure.Storage" Version="9.3.3" />
   </ItemGroup>
+  
+  <IsPackable>true</IsPackable>
+  <IsShipping>false</IsShipping>
```

Comment:
I think if this is set to false, it won't publish the package, which is what we want it to do.

Reply:
Nope, IsShipping being false just means it won't make it to the final sdk product. I think this is correct.

Reply:
Do we actually need to set this? I thought IsShipping is false by default.

Reply:
Arcade documentation suggests otherwise:

> By default all assets produced by a project are considered shipping. Set IsShipping to false if none of the assets produced by the project are shipping. Test projects (IsTestProject is true) set IsShipping to false automatically.

Reply:
Also, This needs to be in a property group

Reply:
IsShipping is not required, you can just do what Arcade does https://github.com/dotnet/arcade-services/blob/master/src/Microsoft.DotNet.Darc/src/Darc/Microsoft.DotNet.Darc.csproj#L8

Reply:
Yes, packages are shipping by default, but 

>IsShipping- properties are project properties that determine which (if any) assets produced by the project are shipping. An asset is considered shipping if it is intended to be delivered to customers via an official channel. This channel can be NuGet.org, an official installer, etc.

Stuff like arcade and Helix sdk are non-shipping. 

I guess the real question is whether darclib itself should be shipping or not. @mmitche I see that darclib is shipping (by lack of the property). Is that by design? If that's by design this should also be shipping, but if these are packages that are only used for internal tooling, they should probably all be non-shipping.

Reply:
We are going to want to use them in the release cli, so we need them to be published nuget packages.

Reply:
So is this good to merge now?

Reply:
They can be published and still be non-shipping (again see arcade and Helix, they are published and flow throughout the repos, but they are still not shipping) . I don't think we need to figure that out in this PR, but we should still figure it out.